### DISCUSSION TITLE
Rsuite  selectpicker virtualized

### DISCUSSION THREAD

**Discussion by Azzmeth** [rsuite/rsuite#3912] [General]
<SelectPicker
                      data={strength_unit}
                      virtualized
                      style={{ width: 300 }}
                    />
![strength](https://github.com/user-attachments/assets/dac63939-1a17-42f7-bee4-aff7a0993ccf)

how i need to solve this

**Comment by simonguo** [comment]
Please reproduce the problem through the [stackblitz example](https://stackblitz.com/edit/react-nm42wy?file=index.js).

**Reply by Azzmeth** [reply]
https://stackblitz.com/edit/react-nm42wy-5ceate?file=index.js

**Reply by simonguo** [reply]
The characters in the options are too long, causing line breaks. When using virtualized, you must fix the line height of each row. You can add a CSS to force the options not to wrap.

```css
.rs-picker-select-menu-item {
  white-space: nowrap;
}
```

**Reply by Azzmeth** [reply]
thank you its  working
